buildconfig/CMake/CppCheckSetup.cmake

```diff
@@ -67,12 +67,12 @@ if ( CPPCHECK_EXECUTABLE )
   # Header files to be ignored require different handling
   set ( CPPCHECK_HEADER_EXCLUDES
         MantidPlot/src/origin/OPJFile.h
-		MantidPlot/src/origin/tree.hh
+        MantidPlot/src/origin/tree.hh
       )
 
   # setup the standard arguments
   set (_cppcheck_args "${CPPCHECK_ARGS}")
-    list ( APPEND _cppcheck_args ${CPPCHECK_TEMPLATE_ARG} )
+    list ( APPEND _cppcheck_args ${CPPCHECK_TEMPLATE_ARG} "--inline-suppr")
```

Comment:
I think `--inline-suppr` is set in [`FindCppcheck.cmake`](https://github.com/mantidproject/mantid/blob/master/buildconfig/CMake/FindCppcheck.cmake#L65) . Does it now need to be set here too?

Reply:
I think this is a leftover from some tests I did and should be reverted. I'll check.